docs/openshift/cert-manager/operator.yaml

```diff
@@ -7,21 +7,20 @@ metadata:
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: cert-manager-operator-og
+  name: openshift-cert-manager-operator
   namespace: cert-manager-operator
 spec:
-  targetNamespaces:
-    - cert-manager-operator
-  upgradeStrategy: Default
+  targetNamespaces: []
+  spec: {}
 ---
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: cert-manager
+  name: openshift-cert-manager-operator
   namespace: cert-manager-operator
 spec:
   channel: stable-v1
   name: openshift-cert-manager-operator
-  installPlanApproval: Automatic
   source: redhat-operators
-  sourceNamespace: openshift-marketplace
\ No newline at end of file
+  sourceNamespace: openshift-marketplace
```

Comment:
In the AWS, it needs role ARN to install openshift cert-manager. So why don't we use community version for cert-manager?

```
Cluster in STS Mode
This cluster is using AWS Security Token Service to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, you will need to provide a role ARN (with an attached policy) during installation. Please see the operator description for more details.
```

Reply:
if then ,it would be a good idea to add some comments about it because the users who have ROSA get confused about the field.